packages/hdwallet-gridplus/src/gridplus.ts

```
@@ -0,0 +1,673 @@
+import { decode as bs58Decode, encode as bs58Encode } from "bs58check";
+import CryptoJS from "crypto-js";
+import {
+  Client,
+  Constants,
+  Utils,
+} from "gridplus-sdk";
+import * as bitcoinjs from "@shapeshiftoss/bitcoinjs-lib";
+import { SignTypedDataVersion, TypedDataUtils } from "@metamask/eth-sig-util";
+import isObject from "lodash/isObject";
+import PLazy from "p-lazy";
+import { encode } from "rlp";
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)